### DISCUSSION TITLE
Problem with Windows 11 and QL operations with Total Commander

### DISCUSSION THREAD

**Discussion by tjsuominen** [QL-Win/QuickLook#1258] [Q&A] [answered]
Running Windows 11 Pro 64-bit.

I have Total Commander installed in "C:\Program Files\Tools\Total Commander" and it works just perfectly.

1. First I tried to use the Windows QL version and installed that from Windows Store
2. Followed the instructions how to integrate QL with TC 

End result: when pressing Alt-F3 in TC > I do not have write access to the QL TC plugin directory

1. Second I used the QL Zip package and manually copied the app files to C:\Program Files\Tools\QuickLook"
2. Followed the instructions...

Error when pressing Alt-F3 : "Data path "C:\Program Files\Tools\QuickLook\UserData" is not writable. Please check if you have sufficient permission".

Are the instructions outdated (I have TC 10.52 64 bit) or am I doing something wrong - what?

**Comment by xupefei** [comment]
Try use the Bridge: https://github.com/QL-Win/QuickLook/wiki/Develop%2C-build-and-integrate#integrating-with-file-managers

**Reply by tjsuominen** [reply]
Ok thx. That can also be used despite separate instructions for Total Commander?

Where exactly this hotkey is expected to be implemented and should the "..."be replaced with %1 or something?

Bridge.exe "c:\a\path\here"

**Comment by tjsuominen** [comment, marked as answer]
Got it working now.

Insstalled QL into TC directory (.msi) and creating the spacebar config to TC. Had to run TC as admin, because the usercmd.ini could not be created to my TC installation directory without admin rights.

**Reply by yardstudio** [reply]
How you have configured command for SPACE?  This is NOT working for me: 
![image](https://github.com/QL-Win/QuickLook/assets/25852179/5a6b222a-4397-4619-b277-0d819331f08a)

I am using portable wesion of TC a nd had unzipped QL archive under USB subfolder 
![image](https://github.com/QL-Win/QuickLook/assets/25852179/2c412b64-9f44-4ab3-bcf3-c4770b1dea03)

**Comment by yardstudio** [comment]
Bridge not working under W10 
![image](https://github.com/QL-Win/QuickLook/assets/25852179/8656f405-2df6-4666-9cd0-25390a244f9c)
